common/server_room.cpp

```diff
@@ -36,6 +36,37 @@ Server_Room::~Server_Room()
     usersLock.unlock();
 }
 
+bool Server_Room::userMayJoin(const ServerInfo_User & userInfo)
+{
+    qDebug() << "USERPRIV: " << QString::fromStdString(userInfo.privlevel());
+
+    ServerInfo_Room roomInfo;
```

Comment:
Whoops left what looks like a debug line in, I'll 🔪 it.  As for the function, its called in the tab_server.cpp file to pull the room permissions to display in the table on the tab.  🍝